compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1228,6 +1228,15 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                     param_names: self.lower_fn_params_to_names(&f.decl),
                 }))
             }
+            TyKind::UnsafeBinder(f) => {
+                let generic_params = self.lower_lifetime_binder(t.id, &f.generic_params);
+                hir::TyKind::UnsafeBinder(self.arena.alloc(hir::UnsafeBinderTy {
+                    span: self.lower_span(t.span),
```

Comment:
this span is redundant with the on on `hir::Ty`, will this be used differently in the future?

Reply:
No, but hir nodes need a span so we gotta store something